examples/README.md

```diff
@@ -1,5 +1,8 @@
 # Sample message files
 
+> [!IMPORTANT]
+> Any new files added to the `examples/Test/` folder should be updated to have `D` in `MSH-11`. New files added to the `examples/Test/Automated/` folder should be updated to have `N` in `MSH-11`
```

Comment:
All example files should have `D` or `N` in them right, not just the ones in `Test`? That matches the changes above

Also the hl7 files in `examples/Test` that _aren't_ in `examples/Test/Automated/` don't appear to have had their MSH-11s updated

Reply:
> All example files should have D or N in them right, not just the ones in Test? That matches the changes above

You are right. It should be for `examples/`

> Also the hl7 files in examples/Test that aren't in examples/Test/Automated/ don't appear to have had their MSH-11s updated

They should be updated. Can you point to which one is not?

Reply:
Fixed path